Status base_status 
Status arm_status
